radio/src/pulses/pulses_arm.h

```diff
@@ -69,28 +69,32 @@ PACK(struct PxxUartPulsesData {
 });
 #endif
 
-/* PXX uses 20 bytes (as of Rev 1.1 document) with 8 changes per byte + stop bit ~= 162 max pulses */
 #if defined(PPM_PIN_TIMER)
+/* PXX uses 20 bytes (as of Rev 1.1 document) with 8 changes per byte + stop bit ~= 162 max pulses */
+/* DSM2 uses 2 header + 12 channel bytes, with max 10 changes (8n2) per byte + 16 bits trailer ~= 156 max pulses */
+/* Multimodule uses 3 bytes header + 22 channel bytes with max 11 changes per byte (8e2) + 16 bits trailer ~= 291 max pulses */
+/* Multimodule reuses some of the DSM2 function and structs since the protocols are similar enough */
+#if defined(MULTIMODULE)
+#define NUM_TIMER_PULSES 300
+#else
+#define NUM_TIMER_PULSES 200
+#endif
+
 PACK(struct PxxTimerPulsesData {
-  pulse_duration_t pulses[200];
+  pulse_duration_t pulses[NUM_TIMER_PULSES];
```

Comment:
For some reason that I don't understand the pulses for PXX only work if the DSM2 part and the pxx part have the same location of the ptr member